cmd/substreams-sink-sql/run.go

```diff
@@ -28,7 +28,9 @@ var sinkRunCmd = Command(sinkRunE,
 		AddCommonSinkerFlags(flags)
 
 		flags.Int("undo-buffer-size", 0, "If non-zero, handling of reorgs in the database is disabled. Instead, a buffer is introduced to only process a blocks once it has been confirmed by that many blocks, introducing a latency but slightly reducing the load on the database when close to head.")
-		flags.Int("flush-interval", 1000, "When in catch up mode, flush every N blocks")
```

Comment:
Can you keep this as deprecated flag and if define, use like if it was `batch-block-flush-interval`, will make upgrade easier.